packages/react-native/scripts/react-native-xcode.sh

```diff
@@ -100,7 +100,13 @@ else
   CONFIG_ARG="--config $BUNDLE_CONFIG"
 fi
 
-BUNDLE_FILE="$CONFIGURATION_BUILD_DIR/main.jsbundle"
+if [[ "$BUNDLE_NAME" ]]; then
+  # Use BUNDLE_NAME defined by user
+  :
+else
+  BUNDLE_NAME="main"
+fi
```

Comment:
can we invert the logic here?
```suggestion
if [[ -z "$BUNDLE_NAME" ]]; then
  BUNDLE_NAME="main"
fi
```

Reply:
Great catch. I had started off by patterning on `ENTRY_FILE` around line 67, but I now realize with the simpler logic here I should have inverted. Thank you!